netci.groovy

```diff
@@ -425,7 +438,8 @@ class Constants {
                'gcstress0xc_zapdisable_heapverify1',
                'gcstress0xc_jitstress1',
                'gcstress0xc_jitstress2',
-               'gcstress0xc_minopts_heapverify1'
+               'gcstress0xc_minopts_heapverify1',
+               'crossgen_comparison'
```

Comment:
this list is sorted identically to the "basicScenarios" + etc. Thus, this should be added after corefx_innerloop, which is missing (!), thus, after "illink".